reana_client/cli/files.py

```diff
@@ -506,30 +506,43 @@ def move_files(ctx, source, target, workflow, access_token):  # noqa: D301
 @check_connection
 @add_access_token_options
 @click.option("-s", "--summarize", is_flag=True, help="Display total.")
+@click.option(
+    "--filter",
+    "filters",
+    multiple=True,
+    help="Filter results to show only file sizes that match certain filtering "
+    "criteria such as file name or size."
+    "Use `--filter <columm_name>=<column_value>` pairs. "
+    "Available filters are `name` and `size`.",
+)
 @human_readable_or_raw_option
 @click.pass_context
 def workflow_disk_usage(
-    ctx, workflow, access_token, summarize, human_readable_or_raw
+    ctx, workflow, access_token, summarize, filters, human_readable_or_raw
 ):  # noqa: D301
     """Get workspace disk usage.
 
     The `du` command allows to chech the disk usage of given workspace.
 
     Examples: \n
     \t $ reana-client du -w myanalysis.42 -s \n
-    \t $ reana-client du -w myanalysis.42 -s --human-readable
+    \t $ reana-client du -w myanalysis.42 -s --human-readable \n
+    \t $ reana-client du -w myanalysis.42 --filter name=workflow
```

Comment:
Minor: I think we could show how to filter directories in this example? So perhaps to use a name like `results` or `data` would be better instead of `workflow`?